proto/01-protoc-base/install.sh

```diff
@@ -23,20 +23,24 @@ python3 -m pip install -q --no-cache-dir --upgrade setuptools
 python3 -m pip install -q --no-cache-dir -r "${__dir}/requirements.txt"
 python3 -m pip list
 
-# Note: should not update past 3.20.x until https://github.com/protocolbuffers/protobuf-javascript/issues/127 is fixed
-
-wget -q "https://github.com/protocolbuffers/protobuf/releases/download/v3.20.3/protoc-3.20.3-linux-x86_64.zip"
+wget -q "https://github.com/protocolbuffers/protobuf/releases/download/v21.2/protoc-21.2-linux-x86_64.zip"
 wget -q "https://repo1.maven.org/maven2/io/grpc/protoc-gen-grpc-java/1.50.2/protoc-gen-grpc-java-1.50.2-linux-x86_64.exe"
+wget -q "https://github.com/protocolbuffers/protobuf-javascript/releases/download/v3.21.2/protobuf-javascript-3.21.2-linux-x86_64.zip"
 sha256sum --check "${__dir}/checksums.txt"
 
-unzip protoc-3.20.3-linux-x86_64.zip -d /opt/protoc
-rm protoc-3.20.3-linux-x86_64.zip
+unzip protoc-21.2-linux-x86_64.zip -d /opt/protoc
+rm protoc-21.2-linux-x86_64.zip
 
 mkdir -p /opt/java/bin
 
 mv protoc-gen-grpc-java-1.50.2-linux-x86_64.exe /opt/java/bin/protoc-gen-grpc-java
 chmod +x /opt/java/bin/protoc-gen-grpc-java
 
+unzip protobuf-javascript-3.21.2-linux-x86_64.zip -d jsplugin_temp
+# move somewhere that is guaranteed to be on the PATH so protoc can find it
+mv jsplugin_temp/bin/protoc-gen-js /usr/local/bin/
+rm -r protobuf-javascript-3.21.2-linux-x86_64.zip jsplugin_temp
+
 mkdir -p /usr/src/app
 cp "${__dir}/package.json" /usr/src/app
 cp "${__dir}/package-lock.json" /usr/src/app
```

Comment:
I wonder if we want to keep the package.json version in lockstep too?

```json
"google-protobuf": "~3.20.1",
```

Reply:
(I don't know how to update package-lock.json...)

Reply:
Done.